### ISSUE TITLE
Colors and font matching to those in the readme

### ISSUE THREAD

**Issue by catliaw** [zakaziko99/agnosterzak-ohmyzsh-theme#17]
Hi @zakaziko99 ,

Thank you for making this theme. This is my first time using oh-my-zsh and your theme caught my eye. I'm just having some trouble getting the font type and colors to match up to those in your screenshots. The coloring is off (and quite blinding), and I can't get the triangle to match up with the line. I installed the fonts suggested. Do you have any idea what the issue may be?

<img width="608" alt="screen shot 2017-07-31 at 1 04 21 pm" src="https://user-images.githubusercontent.com/13739415/28796113-2c40c262-75f1-11e7-88c6-8019c4facbed.png">

**Comment by catliaw** [comment]
I've read the zsh-theme file and installed the Solarized color profiles. I applied the Solarized dark color profile, and the now color issue is slightly closer to your screen shots, but the battery life indicator is missing.

<img width="473" alt="screen shot 2017-07-31 at 1 28 32 pm" src="https://user-images.githubusercontent.com/13739415/28796991-870d7282-75f4-11e7-9aaa-109aad6f0d5e.png">

**Comment by catliaw** [comment]
Not sure if this is intentional, but I figured out that if I have my computer unplugged from power, the battery percentage will show. If it is plugged in, the battery percentage will not show. The coloring is still wonky.

<img width="257" alt="screen shot 2017-07-31 at 1 35 09 pm" src="https://user-images.githubusercontent.com/13739415/28797209-509e6d5e-75f5-11e7-83f9-f65873842dec.png">

Edit: Okay this is intentional!! Sorry to be spamming you. Still trying to fix the coloring issue.

**Comment by catliaw** [comment]
I figured it out. I guess this can remain as documentation for people who are still super noobalicious to zsh and themes. 

<img width="450" alt="screen shot 2017-07-31 at 2 04 11 pm" src="https://user-images.githubusercontent.com/13739415/28798375-2b7f9f12-75f9-11e7-8039-98a99119d72a.png">

In iTerm2, Preferences > Profiles > Text, I changed the font (Change Font) to one of the Powerline fonts that I installed. I chose Meslo LG S Regular for now.

Then I fixed the purple username in the Colors tab. On the right side under ANSI colors, "Magenta" has colors chosen for Normal and Bright. Magenta Bright was the purple color in posts above. I changed it to match Magenta Normal (Hex# d33682).

Hope this helped anyone who is new to themes! :)
